cinema/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for the `MovieSerializer`. This would automatically generate the `create` and `update` methods and simplify the serializer definition by using the `Meta` class to specify the model and fields.